crates/forge/src/test_filter.rs

```diff
@@ -123,7 +123,10 @@ impl TestCaseFilter for TestsFilter {
 
         match self.ignored_filter {
             IgnoredFilter::All => true,
-            IgnoredFilter::Ignored => ignored,
+            IgnoredFilter::Ignored => {
+                assert!(ignored);
+                true
+            }
```

Comment:
Don't think we should be including this change, it's technically true given our current logic, but I think it's safer to still check if the test is ignored here since it's not very expensive.